keyboards/zahranetme/macropad/keymaps/default/config.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2024 Didot <didot404@zlab.my.id>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#ifdef RGB_MATRIX_ENABLE
+#    define RGB_MATRIX_KEYPRESSES
+#endif
```

Comment:
Is there a reason for this existing even though this board doesn't use RGB Matrix, and instead `rgblight` and/or `backlight`?

Edit: Realized this board actually has per-key LEDs on it. Did you forget to add RGB Matrix functionality for this board?

Reply:
it should be rgblight, what needs to be changed?

Reply:
If this board doesn't actually use RGB Matrix despite the per-key LEDs shown in `readme.md` and `rgblight` instead, then you should remove this file and delete any code pertaining to `backlight` in this PR.

Reply:
I only need led underglow